<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Measure</class>
 <widget class="QGroupBox" name="Measure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Measure</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="setFileLabel">
     <property name="text">
      <string>Set File</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="sweepCountLabel">
     <property name="text">
      <string>Sweep Count</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="timeoutLabel">
     <property name="text">
      <string>Timeout</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="dataPathLabel">
     <property name="text">
      <string>Data Path</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="dataPath">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="setFile">
     <item>
      <property name="text">
       <string>&lt;Current Set&gt;</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="sweepCount"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="timeout_s"/>
   </item>
   <item row="4" column="1">
    <widget class="QWidget" name="startWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="start">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="startSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>137</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>setFile</tabstop>
  <tabstop>sweepCount</tabstop>
  <tabstop>timeout_s</tabstop>
  <tabstop>dataPath</tabstop>
  <tabstop>start</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
